packages/docs/docs/self-hosting/opentelemetry.md

```diff
@@ -0,0 +1,116 @@
+---
+sidebar_position: 900
+---
+
+# OpenTelemetry Support
+
+This page describes how to add the Datadog agent to your ECS Fargate tasks. Adding Datadog allows you to collect metrics from all containers.
```

Comment:
Think this line might be leftover copypasta?

Reply:
Eek!  Good catch!